source/tyr/tyr/helper.py

```diff
@@ -348,3 +350,22 @@ def hide_domain(email):
     if "@" not in email:
         return email
     return "{}******".format(email.split('@')[0])
+
+
+def repost_to_another_url(logger, url, content, instance_name):
+    content.seek(0)
+    file_to_post = {"file": (content.filename, content.stream, content.mimetype)}
+    secondary_url = '{}/jobs/{}'.format(url, instance_name)
+    secondary_url = '{}/jobs/ca-qc-sherbrooke'.format(url)
+    nb_try = 0
+    while nb_try < MAX_TRY_FOR_REPOST_TO_SECONDARY_TYR:
+        try:
+            resp = requests.post(secondary_url, files=file_to_post, verify=False)
+            logging.info('Info on posting data: {}'.format(resp.text))
+            if resp.status_code == 200:
+                return True
+            nb_try = nb_try + 1
```

Comment:
if an exception keep occuring inside `requests.post(...)` you will never increase the `nb_try` and you will get stuck forever in the loop. So I think it would be better to increase the counter at the beginning
```suggestion
            nb_try = nb_try + 1
            resp = requests.post(secondary_url, files=file_to_post, verify=False)
            logging.info('Info on posting data: {}'.format(resp.text))
            if resp.status_code == 200:
                return True
            
```

Reply:
my bad, you will return from the function in case of exception :+1: